manifests/r/rcmaehl/MSEdgeRedirect/0.6.4.0/rcmaehl.MSEdgeRedirect.locale.en-US.yaml

```
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: rcmaehl.MSEdgeRedirect
+PackageVersion: 0.6.4.0
+PackageLocale: en-US
+Publisher: rcmaehl

```

```suggestion
Publisher: Robert Maehl Software
```
This must match Control Panel.

---

Sure 😊, will change that